.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "editor.insertSpaces": false,
-    "typescript.tsdk": "node_modules/typescript/lib",
+    "typescript.tsdk": "/Users/mia/Documents/GitHub/pokemon-showdown-dev/node_modules/typescript/lib",
```

Comment:
?

Reply:
Accident, forgot to uncommit that.

Reply:
Was there a problem that required changing this file, or just some tool acting up?